lib/SPIRV/LLVMToSPIRVDbgTran.cpp

```diff
@@ -277,6 +277,9 @@ SPIRVEntry *LLVMToSPIRVDbgTran::transDbgEntryImpl(const MDNode *MDN) {
     case dwarf::DW_TAG_array_type:
       return transDbgArrayType(cast<DICompositeType>(DIEntry));
 
+    case dwarf::DW_TAG_subrange_type:
+      return transDbgSubrangeType(cast<DISubrange>(DIEntry));
```

Comment:
Suggest to add a check for NonSemantic_Kernel_DebugInfo_100 and don't translate the dwarf type if the instruction set is not enabled